rds/sql/dialect/postgreSQL.ts

```diff
@@ -252,6 +257,11 @@ export abstract class AbstractPostgreSqlDialect extends diaImpl.ANSI {
         END;
       $BODY$;
       -- ALTER FUNCTION ${objWfDefnName}${argTypes} OWNER TO (TODO: OWNER);
+      INSERT INTO plpgsql_check_result
+      (function_id, line_no, "statement", sql_state, message, detail, hint,
+       "level", "position", query, context)
+      SELECT functionid, lineno, "statement", sqlstate, message, detail, hint,
+      "level", "position", query, context FROM plpgsql_check_function_tb('${objWfDefnName}');
```

Comment:
@Kunjahamed-P @divyaSkumarCI01188 the `plpgsql_check_function_tb` name should not be hard coded. Use SQL interpolation to fill in the table name based on the entity definition.

Reply:
@shah @Kunjahamed-P 
We will check it and update.